sigmastate/src/main/scala/sigmastate/lang/Types.scala

```diff
@@ -10,12 +11,11 @@ import syntax.Basic.error
 
 //noinspection ForwardReference
 trait Types extends Core {
```

Comment:
ScalaDocs missed in the trait

Reply:
fixed in https://github.com/ScorexFoundation/sigmastate-interpreter/commit/7bfd9e2bc0efb9ab819924d1138d779a2802e86d